src/Toggle/README.md

```diff
@@ -0,0 +1,34 @@
+# Toggle
```

Comment:
Please complete the README.md template: The idea here is explain the props that are part of the component and how we should use them. Keep in mind that this info is used in the Docs tab of storybook.

Reply:
sure, I'll do it today :D